src/downloader.rs

```diff
@@ -105,25 +105,25 @@ impl Downloader {
         match req.send() {
             Ok(mut data) => {
                 let data_type = match data.headers().get("content-type") {
-                    Some(data_type) => data_type.to_str().unwrap(),
-                    None => "text/html",
+                    Some(data_type) => data_type.to_str().unwrap().to_string(),
```

Comment:
`to_str().to_string()`? Isn't there an other way?

Reply:
Nice catch 😅

Reply:
Oh no. data_type is of type ``&HeaderValue`` and I'm using [``to_str()``](https://dtantsur.github.io/rust-openstack/reqwest/header/struct.HeaderValue.html#method.to_str)